src/libraries/System.Net.Http/src/System/Net/Http/HttpClientHandler.cs

```diff
@@ -11,7 +11,7 @@
 using System.Threading.Tasks;
 using System.Diagnostics.Metrics;
 // TODO-LLVM: This is not upstreamable and should be deleted when https://github.com/dotnet/runtimelab/pull/2614 is merged
-#if TARGET_WASI && !NATIVE_AOT
+#if TARGET_WASI
```

Comment:
I presume the TODOs here should be removed?

Reply:
I'm rewriting the whole file upstream to respect child resources and more.

Reply:
I wasn't sure if the `TODO-LLVM` comments were still relevant (i.e. did they refer to just the `&& !NATIVE_AOT` part, or the whole `#if` conditional?)  Sounds like they're no longer relevant, so I'll remove them.

Reply:
There were introduced very recently in https://github.com/dotnet/runtimelab/pull/2605/files. I assumed it was a fix for some build break.